docs/excel2xml-module.md

```diff
@@ -266,7 +266,7 @@ For `make_boolean_prop(cell)`, the following formats are supported:
 - false: False, "false", "False", "0", 0, "no", "No"
 
 N/A-like values will raise an Error. 
-So if your cell is empty, this method will not count it as false, 
+So if your cell is empty, this function will not count it as false, 
```

Comment:
Why do you change it from method to function here and above the reverse? According to this: https://www.shiksha.com/online-courses/articles/difference-between-methods-and-functions-in-python/ currently we do not at all offer methods in `excel2xml-lib`.

Reply:
The change I did was from method -> function, but not the reverse. Function is the correct term.

There is no change in this PR that turns function -> method.